### PR TITLE
Bump vertx-grpc and vertx-core from 4.5.1 to 4.5.3

### PR DESCRIPTION
**Pull request by Madhukar525722**
# :mag: Description

## Describe Your Solution 🔧

Bump vertx-core from 4.5.1 to 4.5.3 to fix CVE-2024-1300 and CVE-2024-1023

## Types of changes :bookmark:

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Test Plan 🧪
Build and ran locally

---

# Checklist 📝

- [ ] This patch was not authored or co-authored using [Generative Tooling](https://www.apache.org/legal/generative-tooling.html)

**Be nice. Be informative.**

### CODE REVIEW

File: dev/dependencyList [hunk 1]
```diff
@@ -182,7 +182,7 @@ swagger-jaxrs2/2.2.1//swagger-jaxrs2-2.2.1.jar
 swagger-models/2.2.1//swagger-models-2.2.1.jar
 trino-client/411//trino-client-411.jar
 units/1.7//units-1.7.jar
-vertx-core/4.5.1//vertx-core-4.5.1.jar
+vertx-core/4.5.3//vertx-core-4.5.3.jar
```
Comment (cxzl25) [inline dev/dependencyList:185]: Only updating the `vertx-core` version may not guarantee compatibility with `vertx-grpc`.
The vertx dependency is introduced by jetcd. How about directly upgrading the jetcd version?

Comment (Madhukar525722) [reply dev/dependencyList:185]: HI @cxzl25 , I tried upgrading jetcd to 0.8+, but it gave the error and asked to have java 11+
I agree to you we can try upgrading vertx-grpc itself

File: pom.xml [hunk 2]
```diff
@@ -909,6 +909,12 @@
                 </exclusions>
             </dependency>
 
+            <dependency>
+                <groupId>io.vertx</groupId>
+                <artifactId>vertx-grpc</artifactId>
+                <version>4.5.3</version>
```
Comment (bowenliang123) [inline pom.xml:915]: Extract the `4.5.3` to a new property `vertx.version` in `<properties>` node.

**Review by bowenliang123 — APPROVED** [review]

**Review by cxzl25 — APPROVED** [review]

**Comment by bowenliang123** [comment]
Thanks for your contribution. Merged to master (1.10.0).

**Pull request closed without merging**